metadeploy/api/migrations/0123_remove_user_site.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.13 on 2022-06-09 23:34
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("api", "0122_more_multi_tenancy_models"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="user",
+            name="site",
+        ),
+    ]
```

Comment:
@jerivas Can we add a reverse here so this can be un-applied?

> django.db.utils.IntegrityError: column "site_id" contains null values

Reply:
The problem is that [`RemoveField` is irreversible if the field is not nullable](https://docs.djangoproject.com/en/4.0/ref/migration-operations/#removefield). I went ahead and [squashed the migrations](https://github.com/oddbird/MetaDeploy/commit/8bce3376a2f46ce8642a333d2fd0d5da228b6406) instead to completely sidestep the issue. In my test the DB correctly picked up that I didn't need to run the new squashed migration because the table columns were up to date